### PR TITLE
Add methods for other export formats.

### PR DESCRIPTION
**Pull request by manuelesvi**
A new method was added to export the Graph using PNG format as well.

### CODE REVIEW

File: Rubjerg.Graphviz/Graph.cs [hunk 1]
```diff
@@ -483,14 +483,27 @@ public void FreeLayout()
                 throw new ApplicationException($"Graphviz render returned error code {free_rc}");
         }
 
+        private void RenderToFile(string filename, string format)
```
Comment (manuelesvi) [inline Rubjerg.Graphviz/Graph.cs:486]: This is the original code which was moved to a private method to reuse it from both callers.

File: Rubjerg.Graphviz/Graph.cs [hunk 2]
```diff
@@ -483,16 +483,31 @@ public void FreeLayout()
...
 
+        /// <summary>
+        /// Should only be called after <see cref="ComputeLayout"/> has been called.
+        /// </summary>
+        public void ToPngFile(string filename)
+        {
+            RenderToFile(filename, "png");
+        }
+
+        public void ToPdfFile(string filename) => RenderToFile(filename, "pdf");
```
Comment (chtenb) [inline Rubjerg.Graphviz/Graph.cs:509]: Can you add the same docstring to this methods as the others?

**Comment by chtenb** [comment]
Looks good to me, with the one comment.

File: Rubjerg.Graphviz/Graph.cs (hunk 2 above)
Comment (manuelesvi) [reply Rubjerg.Graphviz/Graph.cs:512]: Surely, there you go.

**Pull request merged**